src/deploy/RPM_build/noobaa.spec

```diff
@@ -31,11 +32,30 @@ NooBaa is a data service for cloud environments, providing S3 object-store inter
 mkdir noobaa-core-%{version}-%{revision}
 mkdir node-%{nodever}
 tar -xzf %{SOURCE0} -C noobaa-core-%{version}-%{revision}/
-tar -xJf %{SOURCE1} -C node-%{nodever}/
 
 %clean
 [ ${RPM_BUILD_ROOT} != "/" ] && rm -rf ${RPM_BUILD_ROOT}
 
+%build
+#!/bin/bash
+NOOBAA_DIR="noobaa-core-%{version}-%{revision}"
+NODEJS_VERSION="%{nodever}"
+SKIP_NODE_INSTALL=1 source $NOOBAA_DIR/noobaa/src/deploy/NVA_build/install_nodejs.sh $NODEJS_VERSION
+
+pushd ~/rpmbuild/SOURCES/
+nodepath=$(download_node)
+tar -xJf ${nodepath} -C ~/rpmbuild/BUILD/node-%{nodever}/
+popd
+
+PATH=$PATH:$HOME/rpmbuild/BUILD/node-%{nodever}/node-v$NODEJS_VERSION-linux-$(get_arch)/bin
+
+pushd $NOOBAA_DIR/noobaa
+npm install --omit=dev && npm cache clean --force
+./src/deploy/NVA_build/clone_s3select_submodules.sh
+ln -s /lib64/libboost_thread.so.1.66.0 /lib64/libboost_thread.so.1.75.0 || true
```

Comment:
Why do we need this? This looks like an issue with rhel 8 vs rhel 9 boost version. Can we use the proper rhel image instead?

Reply:
Actually the base image can be any of RH8 or RH9. I borrowed (to be fair, copied) it from our Dockerfile.

Reply:
Well, this line updates the underlying system, i.e. it updates the builder machine, not the machine where it will be installed.

Reply:
Addressed in the latest commit.